nuxt.config.js

```diff
@@ -24,7 +24,25 @@ export default {
   css: [],
 
   // Plugins to run before rendering page: https://go.nuxtjs.dev/config-plugins
-  plugins: [{ src: "~/plugins/vue-globals", ssr: true }], 
+  plugins: ['@/plugins/mirage.js', { src: '@/plugins/vue-globals', ssr: true }],
+
+  // https://nuxtjs.org/blog/moving-from-nuxtjs-dotenv-to-runtime-config
+  publicRuntimeConfig: {
+    baseURL: process.env.NUXT_ENV_BASE_URL || 'http://localhost:3000',
+  },
+
+  ssr: process.env.NUXT_ENV_USE_MIRAGE === 'true' ? false : true, // Disable Server Side rendering for development because of miragejs
```

Comment:
question (blocking): Can't you just write `process.env.NUXT_ENV_USE_MIRAGE === 'true' ` ?